clang/test/CodeGen/complex.c

```diff
@@ -113,3 +113,8 @@ void t92(void) {
   (0 ? (_Complex double) 2.0f : 2.0f);
 }
 
+// PR69218
+int t10(void) {
+  float _Complex a;
+  return (0 < &__real__ a) && (0 < &__imag__ a);
+}
```

Comment:
I think this test should live in `clang/test/Sema/complex-imag.c` because the original crash doesn't require codegen to run to see it (it happens with `-fsyntax-only`).